notes/linuxdev/index.md

```diff
@@ -28,7 +28,7 @@ Configure all of your settings in the Build & Run section.
    - Under **Deployment** click **Add Deploy Step &gt; Make**, and in **Make arguments** add "install".  
    - Under **Run &gt; Run configuration** click **Add &gt; Custom Executable &gt; Executable** add   `$HOME/Projects/Shotcut/Shotcut.app/bin/shotcut`
    - Under **Run Environment &gt; Details** add the following environment variables:
-     - Set `LD_LIBRARY_PATH` to `${HOME}/Projects/Shotcut/Shotcut.app/lib:${QTDIR}/lib:${HOME}/Projects/Shotcut/Shotcut.app/src/build-shotcut-Desktop_Qt_5_6_1_GCC_64bit-Debug/CuteLogger`
+     - Set `LD_LIBRARY_PATH` to `${HOME}/Projects/Shotcut/Shotcut.app/lib:${QTDIR}/lib:${HOME}/Projects/Shotcut/Shotcut.app/src/build-shotcut-Desktop_QT-Debug/CuteLogger`
```

Comment:
This path change is not correct for the way my Qt Creator creates per the Kit name by default based on the Qt version. Mine is `build-shotcut-Desktop_Qt_5_9_7_GCC_64bit-Debug`.

Reply:
Whoops - my bad, sorry for the noise. I had indeed truncated it during import into QTcreator